packages/twenty-front/src/pages/settings/workspace/SettingsDomain.tsx

```diff
@@ -101,7 +101,8 @@ export const SettingsDomain = () => {
     } catch (error) {
       if (
         error instanceof Error &&
-        error.message === 'Subdomain already taken'
+        (error.message === 'Subdomain already taken' ||
+          error.message.endsWith('not allowed'))
```

Comment:
this feels very manual! We will need to discuss later if we actually want the backend to return error codes to the FE